examples/liquid.ini

```diff
@@ -217,6 +217,13 @@ wait_green_count = 6
 #                         Set to "-1" for batch processing (indexing), or "6s" for normal searching.
 #                         Defaults to "6s".
 #                         Any collection can override this value.
+# - `image_classification_enabled` - Set to "true" to run image classification service. The service can perform
```

Comment:
I don't think we need this boolean to be supplied by us, since you can just do an `or` between `image_classification_object_detection_enabled` and `image_classification_classify_images_enabled`, right?